app/search_builders/scholars_archive/facet_modal_search_builder.rb

```diff
@@ -0,0 +1,41 @@
+module ScholarsArchive
+  class FacetModalSearchBuilder
+    def admin_records(facet_string)
+      "(" + [facet_search_string(facet_string), not_filesets].join(" AND ") + ")"
```

Comment:
Nitpicky, for sure.. but from a readability perspective could you use string interpolation for these methods instead of concatenation?

```
clauses = [facet_search_string(facet_string), not_filesets].join(" AND ")
"(#{clauses})"
```